docker-java/src/test/java/com/github/dockerjava/cmd/LoadImageCmdIT.java

```diff
@@ -53,6 +54,19 @@ public void loadImageFromTar() throws Exception {
                 asList(image.getRepoTags()), equalTo(singletonList("docker-java/load:1.0")));
     }
 
+    @Test
+    public void loadImageFromTarAsync() throws Exception {
```

Comment:
Shouldn't we also have a successful test?

Reply:
you mean unsuccessful, right?

Reply:
Never mind, like always, I read the descriptions of the `assertThat` method wrongly -_-